### PR TITLE
Misc. updates

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Probably see #2154 first.

- Config flags description update
- SSH URL updates
- Term typo in GS->Pipelines per https://github.com/iterative/dvc.org/issues/2170#issuecomment-776486084

### CODE REVIEW

File: content/docs/user-guide/external-dependencies.md [hunk 1]
```diff
@@ -88,7 +91,7 @@ $ dvc run -n download_file \
 $ dvc run -n download_file \
           -d ssh://user@example.com/path/to/data.txt \
           -o data.txt \
-          scp user@example.com:/path/to/data.txt data.txt
+          scp ssh://user@example.com:/path/to/data.txt data.txt
```
Comment (shcheklein) [inline content/docs/user-guide/external-dependencies.md:94]: hmm, I don't usually specify the protocol like this

**Review by shcheklein — APPROVED** [review]
some minor questions, typos

File: content/docs/command-reference/remote/modify.md [hunk 2]
```diff
@@ -500,7 +505,8 @@ more information.
 ### Click for SSH
 
 - `url` - remote location, in a regular
-  [SSH format](https://tools.ietf.org/id/draft-salowey-secsh-uri-00.html#sshsyntax):
+  [SSH format](https://tools.ietf.org/id/draft-salowey-secsh-uri-00.html#sshsyntax).
+  Note that this can already the `user` parameter, embedded into the URL:
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md:509]: Ah yes, "included". Thanks

File: content/docs/user-guide/external-dependencies.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/external-dependencies.md:94]: Right. Reverting this change.

**Pull request merged**